--- a/src/myconfig.c
+++ b/src/myconfig.c
@@ -49,7 +49,7 @@ static const char *PACKAGE_BUGREPORT = "
 #ifdef MAC_OS
 static const char *D_DHCPSCRIPT = "dhcping -v -t 15";	/* 默认DHCP脚本 */
 #else
-static const char *D_DHCPSCRIPT = "dhclient";	/* 默认DHCP脚本 */
+static const char *D_DHCPSCRIPT = "udhcpc -i";	/* 默认DHCP脚本 */
 #endif
 static const char *CFG_FILE = "/etc/mentohust.conf";	/* 配置文件 */
 static const char *LOG_FILE = "/tmp/mentohust.log";	/* 日志文件 */
@@ -653,7 +653,7 @@ static void showHelp(const char *fileNam
 #ifndef NO_GETOPT_LONG
         "\t--dhcp-script"
 #endif
-		"\t-c DHCP脚本[默认dhclient]\n"
+		"\t-c DHCP脚本[默认udhcpc]\n"
 #ifndef NO_GETOPT_LONG
         "\t--decode-config"
 #endif
